src/mscorlib/src/System/Text/Encoding.cs

```diff
@@ -1230,10 +1230,8 @@ private static Encoding CreateDefaultEncoding()
 
             Encoding enc;
 
-
-            // For silverlight we use UTF8 since ANSI isn't available
-            enc = UTF8;
-
+            // For netcore we use UTF8 since ANSI isn't available
+            enc = new UTF8Encoding.UTF8EncodingSealed(encoderShouldEmitUTF8Identifier: false);
```

Comment:
Rather than initializing via an interlocked; would it be easier to make the type fixed and more visible for devirtualization; as per UTF8Encoding?

```csharp
private static readonly UTF8EncodingSealed s_defaultEncoding
  = new UTF8EncodingSealed(encoderShouldEmitUTF8Identifier: false);
public static Encoding Default => s_defaultEncoding;
```

Since its the only static var in the class it shouldn't be initialized when its not used? (e.g. some other static var is accessed which causes its initialization)